### PR TITLE
Recall msg

### PR DESCRIPTION
**Pull request by SilentQianyi**
## I'm submitting a...

```
[ ] Bug Fix
[ ] Feature
[ ] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

- [ ] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [ ] Tests for the changes have been added

## Description

_please describe the changes that you are making_

_for features, please describe how to use the new feature_

_please include a reference to an existing issue, if applicable_

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

Issue
https://github.com/wechaty/wechaty/issues/1885

### CODE REVIEW

File: src/user/message.ts [hunk 1]
```diff
@@ -566,6 +567,32 @@ export class Message extends Accessory implements Sayable {
...
+   * Recall a message.
+   * > Tips:
+   * @returns {Promise<boolean>}
+   *
+   * @example
+   * const bot = new Wechaty()
+   * bot
+   * .on('message', async m => {
+   *   const recallMessage = await msg.say('123')
+   *   const isSuccess = await recallMessage.recall()
```
Comment (CooperFu) [inline src/user/message.ts]: recall should need a messageId.

Comment (windmemory) [reply src/user/message.ts]: I don't think so, this is a function on `Message` class, it is recalling the message which called this function.

Comment (windmemory) [reply src/user/message.ts]: I don't think so, this is a function on `Message` class, it is recalling the message which called this function.

Comment (CooperFu) [reply src/user/message.ts]: ok then, I send two messages. what should I do for I want to recall the first one?
```
const recallMessage = await msg.say('123')
const recallMessage2 = await msg.say('456')
const isSuccess = await recallMessage.recall()
```
like this?

Comment (SilentQianyi) [reply src/user/message.ts]: yes.

File: src/user/message.ts [hunk 2]
```diff
@@ -566,6 +566,24 @@ export class Message extends Accessory implements Sayable {
...
```
Comment (huan) [inline src/user/message.ts]: `recallMessage` might be null, please fix the example code for this.

**Pull request merged**